SerialPrograms/Source/Integrations/DppIntegration/DppUtility.cpp

```diff
@@ -58,6 +58,27 @@ uint16_t Utility::get_button(const uint16_t& bt) {
     return 1 << bt;
 }
 
+int64_t Utility::get_value_from_input(const commandhandler& handler, const std::string& command_name, const std::string& input, std::string& out) {
+    auto cmd = handler.commands.find(command_name);
+    auto& choices = cmd->second.parameters[1].second.choices;
```

Comment:
No chance `cmd->second.parameters` has fewer than 2 elements right?

Reply:
`commandhandler` expects you to know the size and parameter types. If you supply empty parameters, it will fail the `parameter.size()` check. If you provide at least one parameter, the rest get initialized as defaults. In other words, this should (surprisingly) be safe.

Reply:
In this particular case, I'm iterating through what we have generated ourselves, so it wouldn't fail anyways. Bu, again, we're expected to know this.